sdk/compute/Azure.ResourceManager.Compute/src/autorest.md

```diff
@@ -10,7 +10,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 azure-arm: true
 library-name: Compute
 namespace: Azure.ResourceManager.Compute
-require: https://github.com/Azure/azure-rest-api-specs/blob/ce47f9b775ec53750f37def0402ecacf3f1d661b/specification/compute/resource-manager/readme.md
+require: /mnt/vss/_work/1/s/azure-rest-api-specs/specification/compute/resource-manager/readme.md
```

Comment:
Do I need to keep this format? Below is what it would be.
https://github.com/Azure/azure-rest-api-specs/blob/39c1eb0bb8512315a35fc726c72895d131775253/specification/compute/resource-manager/readme.md